source/Projectile.cpp

```diff
@@ -73,17 +73,19 @@ Projectile::Projectile(const Projectile &parent, const Point &offset, const Angl
 	if(inaccuracy)
 	{
 		this->angle += Angle::Random(inaccuracy) - Angle::Random(inaccuracy);
-		if(!parent.weapon->Acceleration())
-		{
-			// Move in this new direction at the same velocity.
-			// To maintain the sign of the velocity, Point::Length can’t be used.
-			Point referenceVector = parent.angle.Unit();
-			double parentVelocity = referenceVector.Dot(parent.velocity);
-			velocity = this->angle.Unit() * parentVelocity;
-		}
 	}
-	velocity += this->angle.Unit() * (weapon->Velocity() + Random::Real() * weapon->RandomVelocity());
 
+	// Start with the parent's current velocity
+	velocity = parent.velocity;
+	
+	// Revert to the velocity of the parent projectile.
+	velocity -= parent.angle.Unit() * (parent.GetWeapon().Velocity() + (Random::Real() * parent.GetWeapon().RandomVelocity()));
```

Comment:
This `(Random::Real() * parent.GetWeapon().RandomVelocity())` won't return the same random value as when the projectile was first created, so you're not actually getting what you want.

Reply:
I'm aware of that but I don't believe any random velocity value is carried through from firing.  Even still; we could use half of the random velocity stat instead of randomizing it to the full value like I've done.  Or we could ignore it but I wouldn't do that.

Reply:
I changed it to use half of the random velocity of the parent every time instead.  I'm believe that's used as a double and not an int, but either way the net speed will be the same.  That is, there may be a very slight difference in how much inaccuracy is applied for the submunition but the same amount of speed that is subtracted will always be added back.